kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -304,12 +541,30 @@
 
   /**
    * Delete a {@link Dataset} identified by the given dataset URI.
```

Comment:
This isn't quite correct because view URIs are supported. It should be "Delete a Dataset or View identified by the given dataset URI."